helpers/sqs_helper.py

```diff
@@ -0,0 +1,108 @@
+"""
+Helper module for sqs messages
+"""
+import json
+import os
+import sys
+import time
+import boto3
+from botocore.exceptions import ClientError
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers.base_helper import BaseHelper
+from conf import sqs_conf
+
+class SqsHelper(BaseHelper):
```

Comment:
This should be a class. But the way it has been implemented here, it might as well been a module with methods.